hasura/metadata/databases/default/tables/public_users.yaml

```diff
@@ -2,181 +2,181 @@ table:
   name: users
   schema: public
 object_relationships:
-- name: circle
```

Comment:
please make sure you're using the same hasura version as in package.json, 2.2.0. otherwise it creates these indentation changes. (we'll want to upgrade this version of hasura and check in just indentation changes soon, in a different PR.)

Reply:
please re-export the metadata once you are using the right version; the diffs should become a lot smaller